index.html

```diff
@@ -21,34 +23,225 @@
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="card-tools-list">
+            <li class="card-tools">html</li>
+            <li class="card-tools">css</li>
+            <li class="card-tools">javascript</li>
+          </ul>
+          <nav class="card-button-box">
+            <button class="card-button">See Project</button>
+          </nav>
+        </div>
+
+        <!-- Card 4 -->
+        <div class="card-works-container card-size1">
+          <img src="./image/Snapshoot-Portfolio-Art-Print.png" alt="Project4">
+          <h1 class="card-title-text">Multi-Post Stories</h1>
+          <div class="card-list">
+            <span class="card-canopy">
+              CANOPY
+            </span>
+            <div class="card-bullet-separator"></div>
+            <span class="card-canopy card-canopy-details">
+              Back End Dev
+            </span>
+            <div class="card-bullet-separator"></div>
+            <span class="card-canopy card-canopy-details">
+              2015
+            </span>
+          </div>
+          <p class="card-main-text">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="card-tools-list">
+            <li class="card-tools">html</li>
+            <li class="card-tools">css</li>
+            <li class="card-tools">javascript</li>
+          </ul>
+          <nav class="card-button-box">
+            <button class="card-button">See Project</button>
+          </nav>        
+        </div>
+      </section>
+
+      <!-- About Myself -->
+      <section class="about-myself">
+        <h1 id="about-myself-header">About <br />Myself</h1>
+        <p class="about-myself-description">
+          Hello I’m a software developer! I can help you build a product , feature or website. 
+          Look through some of my work and experience! If you like what you see and have a 
+          project you need coded, don’t hesitiate to contact me.
+        </p>
+        <h6 class="connect">
+          LET’S CONNECT
+        </h6>
+        <ul class="iconlists">
+          <li>
+            <a href="https://github.com/"><img src="./image/Vector 1.png" alt="" /></a>
+          </li>
+          <li>
+            <a href="https://linkedin.com/"><img src="./image/Linkedin icon.png" alt="" /></a>
+          </li>  
+          <li>
+            <a href="https://shutterstock.com/"><img src="./image/Vector 3.png" alt="" /></a>
+          </li>
+          <li>            
+            <a href="https://twitter.com/"><img src="./image/Twitter.png" alt="" /></a>
+          </li>
+          <li>
+            <a href="https://medium.com/"><img src="./image/Vector 2.png" alt="" /></a>
+          </li>                    
+        </ul>        
+        <nav class="card-button-box">
+          <button class="card-button">Get my resume</button>
+        </nav>
```

Comment:
- [ ] - [ OPTIONAL ]  Everything is fine here but I don't think it is the best way to use the nav tag in the middle of the page just as a container, rather you use a div, please make sure you modify it later.

Reply:
@joskalenda I have replaced the nav tag with a div tag. Thanks for the feedback and the tips.